src/cdk/a11y/key-manager/tree-key-manager.ts

```diff
@@ -106,6 +106,8 @@ export class TreeKeyManager<T extends TreeKeyManagerItem> implements TreeKeyMana
 
       this._setTypeAhead(typeAheadInterval);
     }
+
+    this._focusInitialNode();
```

Comment:
instead of calling this in constructor, try calling the first time we receive items that are not empty.

- query: call in subscription
- observable: call in subscription
- array: call directly